misc/acrn-config/schema/config.xsd

```diff
@@ -251,7 +252,7 @@
       </xs:annotation>
     </xs:element>
     <xs:element name="guest_flags" type="GuestFlagsInfo" minOccurs="0">
-      <xs:annotation>
+      <xs:annotation acrn:multiselect="true">
```

Comment:
Do we change acrn:multiselect="true" to acrn:multiselect="y" according to previous consistent rules for boolean values?

Reply:
@szhen11 Yes